docs/source/usage.rst

```diff
@@ -305,6 +305,12 @@ Input yaml file:
     >>> print(conf.user.home)
     /home/omry
 
+It is possible to set a default value if the environment variable is missing. 
+In the following example, we set 'omry' as the default value if the environment
+ variable USER is not defined.
```

Comment:
1. USER is not the best example because it is define by default. use a different env name for this example.
2. Show a code example here, this is the actual documentation, it's more important than Jupyter notebook.
You can just make it all inline, no need for an additional file:
```python
cfg = OmegaConf.create({
   "database": {
      password: "${env:DB_PASSWORD,12345}"
   }
})

print(cfg.database.password)
12345

os.environ["DB_PASSWORD"] = secret
print(cfg.database.password)
secret
```

Reply:
Example added. Had to put in `cfg = cfg.copy()` to clear resolver catch.